.github/workflows/auto-pr.yml

```diff
@@ -1,7 +1,7 @@
 name: Extension issue to Pull Request conversion
 on:
   issues:
-    types: [opened, labeled, edited]
+    types: [labeled, edited]
```

Comment:
Fixing the issue where GitHub Actions are triggered at once with 'opened' and 'labeled' conditions when an issue is opened.

Reply:
The issue arises when the '✨ New extension' label is automatically applied as the issue is opened.